File: BackendPhotoItemsApp/Models/ItemPhoto.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackendPhotoItemsApp.Models
{
    public partial class ItemPhoto
    {
        public ItemPhoto()
        {
            ItemPhotoPropertySets = new HashSet<ItemPhotoPropertySet>();
        }

        public int Id { get; set; }
        public int? ItemId { get; set; }
        public int TypeId { get; set; }
        public string FileName { get; set; }
        public int? Position { get; set; }
        public string Alt { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? ModifiedAt { get; set; }
        public bool? IsActive { get; set; }

        public virtual Item Item { get; set; }
        public virtual Type Type { get; set; }
        public virtual ICollection<ItemPhotoPropertySet> ItemPhotoPropertySets { get; set; }
    }
}

File: frontend-photo-items-app/src/models/ItemPhoto.ts
import Item from "./Item";
import ItemPhotoPropertySet from "./ItemPhotoPropertySet";
import Type from "./Type";

export default class ItemPhoto {

  public id: number = 0;
  public itemId: number = 0;
  public typeId: number = 0;
  public fileName: string = "";
  public position: number = 0;
  public alt: string = "";
  public createdAt: Date = new Date();
  public modifiedAt: Date = new Date();
  public isActive: boolean = true;

  //Navigations
  public itemNavigation: Item = new Item();
  public typeNavigation: Type = new Type();
  public itemPhotoPropertySet: ItemPhotoPropertySet[] = [];

  constructor(itemPhoto?: Partial<ItemPhoto>) {
    Object.assign(this, itemPhoto);
  }

}